docs/snapshots.rst

```diff
@@ -0,0 +1,75 @@
+.. -*- coding: utf-8 -*-
+
+.. _snapshots:
+
+
```

Comment:
Perhaps we could introduce the purpose of this document here, including who the intended audience is (developers, I guess).